components/input/input.css

```diff
@@ -0,0 +1,24 @@
+.input {
+  background: transparent;
+  color: var(--text-color);
+  border: 1px solid transparent;
```

Comment:
You can use `border: none;` or `border:0` as you use an outline.

Reply:
Done